### PR TITLE
Sukh player collision

### PR DESCRIPTION
**Pull request by ssidhu271**
Got collision working between the enemy and the player. When the Enemy and player hit enemy is removed from the map and player will take damage.

### CODE REVIEW

File: src/main/java/org/bcit/comp2522/project/Window.java [hunk 1]
```diff
@@ -14,224 +12,237 @@
...
+                || projectile.getPosition().y < 0 || projectile.getPosition().y > height;
+        if (toRemove) {
+          collectionManager.getSprites().remove(projectile);
+        }
+        return toRemove;
+      });
+      ArrayList<Enemy> toRemove = new ArrayList<>();
+      ArrayList<Projectile> projectilesToRemove = new ArrayList<>();
+      for (Enemy enemy : collectionManager.getEnemies()) {
+        if (enemy.checkCollisionWithPlayer(CollectionManager.player)) {
```
Comment (OzanYRT) [inline src/main/java/org/bcit/comp2522/project/Window.java:195]: This is the code ended up changing, please review Brett. The other changes were minor changes that you guys agreed on too.

**Review by OzanYRT — APPROVED** [review]
Looks good overall. Please review Sukhs changes in Window class draw method.

File: src/main/java/org/bcit/comp2522/project/Window.java (hunk 1 above)
Comment (breader1) [reply src/main/java/org/bcit/comp2522/project/Window.java:195]: This code is okay for now. Sukh and I discussed moving the toRemove Collections to CollectionManager.java in the near future

File: src/main/java/org/bcit/comp2522/project/SaveHandler.java [hunk 2]
```diff
@@ -16,7 +16,7 @@ private void save() {
   public void autoSave(){
     for (; ; ) {
       try {
-        Thread.sleep(10000);
+        Thread.sleep(30000);
```
Comment (breader1) [inline src/main/java/org/bcit/comp2522/project/SaveHandler.java:19]: lol. Based

**Review by breader1 — APPROVED** [review]

**Pull request merged**